### PR TITLE
Add missing extension points

### PR DESCRIPTION
**Pull request by CGNonofr**

### CODE REVIEW

File: src/missing-services.ts [hunk 1]
```diff
@@ -1732,7 +1732,21 @@ registerSingleton(INotebookEditorModelResolverService, class NotebookEditorModel
 registerSingleton(IWorkingCopyEditorService, WorkingCopyEditorService, InstantiationType.Delayed)
 registerSingleton(IUserActivityService, UserActivityService, InstantiationType.Delayed)
 registerSingleton(ICanonicalUriService, CanonicalUriService, InstantiationType.Delayed)
-registerSingleton(IExtensionStatusBarItemService, ExtensionStatusBarItemService, InstantiationType.Delayed)
+registerSingleton(IExtensionStatusBarItemService, class ExtensionStatusBarItemService implements IExtensionStatusBarItemService {
```
Comment (CompuIves) [inline src/missing-services.ts:1735]: This is interesting, I believe we're already running extensions that contribute status bar items. Does this add something else as well?

**Review by CompuIves — APPROVED** [review]
Straightforward PR!

File: src/missing-services.ts (hunk 1 above)
Comment (CGNonofr) [reply src/missing-services.ts:1735]: I've just moved the implementation into the dedicated service override, no change expected here

Comment (CGNonofr) [reply src/missing-services.ts:1735]: Maybe you're not talking about this specific line but the new `StatusBarItemsExtensionPoint`? then yes probably new things to expect but I don't really know why (:

**Pull request merged**